biomage/rotate_ci/rotate_ci.py

```diff
@@ -275,10 +278,23 @@ def rotate_ci(token, org):
     repos = exclude_iac_from_rotation(repos, org.login)
 
     policies = [ret for ret in (filter_iam_repos(repo) for repo in repos) if ret]
+    policies = dict(policies)
+
     click.echo(
-        f"Found {len(policies)} repositories marked as requiring CI IAM policies."
+        f"Found {len(policies.keys())} repositories marked as requiring CI IAM policies.\nThese are: {policies.keys()}" 
     )
-    policies = dict(policies)
+
+    questions = [
+            Confirm(
+                name="create",
+                message="Are you sure you want to rotate ci for these repositories?",
+                default=False,
+            )
+        ]
+    click.echo()
+    answer = prompt(questions, theme=GreenPassion())
```

Comment:
print the repos and ask before rotating the keys, just in case something is wrong.